diff -Naur nfs-utils-2.3.3/utils/mount/network.c nfs-utils-2.3.3-new/utils/mount/network.c
--- nfs-utils-2.3.3/utils/mount/network.c	2018-09-06 11:09:08.000000000 -0700
+++ nfs-utils-2.3.3-new/utils/mount/network.c	2019-04-09 02:59:50.199109284 -0700
@@ -39,7 +39,7 @@
 #include <sys/socket.h>
 #include <sys/wait.h>
 #include <sys/stat.h>
-#if defined(__GLIBC__) && (__GLIBC__ < 2) || (__GLIBC__ == 2 && __GLIBC_MINOR__ < 24)
+#if defined(__GLIBC__) && (__GLIBC__ < 2) || (__GLIBC__ == 2 && __GLIBC_MINOR__ < 25)
 /* Cannot safely include linux/in6.h in old glibc, so hardcode the needed values */
 # define IPV6_PREFER_SRC_PUBLIC 2
 # define IPV6_ADDR_PREFERENCES 72
