diff -Naur mgetty-1.1.32/logfile.c mgetty-1.1.32.new/logfile.c
--- mgetty-1.1.32/logfile.c	2004-11-08 12:07:29.000000000 -0800
+++ mgetty-1.1.32.new/logfile.c	2005-04-06 00:41:20.132544967 -0700
@@ -330,9 +330,7 @@
 	fprintf(log_fp, "\n%02d/%02d %02d:%02d:%02d %s %s: %s",
 		             tm->tm_mon+1,  tm->tm_mday,
 			     tm->tm_hour, tm->tm_min, tm->tm_sec,
-		             log_infix, ws,
-			     ( errnr <= sys_nerr ) ? sys_errlist[errnr]:
-			     "<error not in list>" );
+		             log_infix, ws,strerror(errnr));
 #ifdef SYSLOG
 	syslog( level == L_FATAL? LOG_ALERT: LOG_ERR, "%s: %m", ws );
 #endif
