diff -Naur mgetty-1.2.1.org/mgetty.h mgetty-1.2.1/mgetty.h
--- mgetty-1.2.1.org/mgetty.h	2020-09-17 20:21:46.279391452 -0700
+++ mgetty-1.2.1/mgetty.h	2020-09-17 20:29:09.339673727 -0700
@@ -218,7 +218,7 @@
  * union to use "void *" and "long", instead of "int" (see config.h).
  * Same for Sparc Ultra machines [at least with SparcLinux]
  */
-#if defined(__alpha__) || defined(__sparc64__) || \
+#if defined(__aarch64__) || defined(__alpha__) || defined(__sparc64__) || \
 	    defined(__ia64__) || defined(__s390x__) || defined(__x86_64__) || \
 	    defined(__powerpc64__)
 # define PTR_IS_LONG
